crypto/openstark/src/proofs.rs

```diff
@@ -413,10 +403,10 @@ fn get_constraint_polynomials(
     // Convert to even and odd coefficient polynomials
     let mut constraint_polynomials: Vec<Vec<FieldElement>> =
         vec![
-            Vec::with_capacity(trace_coset.num_rows() / constraints_degree_bound);
-            constraints_degree_bound
+            Vec::with_capacity(trace_coset.num_rows() / trace_degree(constraints));
+            trace_degree(constraints)
         ];
-    for chunk in values.chunks_exact(constraints_degree_bound) {
+    for chunk in values.chunks_exact(trace_degree(constraints)) {
```

Comment:
We are recomputing this value quite often. We can keep it around in a variable, which I think would make sense here, in `proof` and in `verifier`.